test/unit/ccl/functions/__analyzeDccWallet.spec.js

```diff
@@ -193,6 +195,56 @@ End of debugging: ${chalk.magenta(testCaseDescription)}`
                 )
               })
             })
+
+            context('certificateReissuance', () => {
+              const {
+                certificateReissuance: expCertificateReissuance
+              } = assertions
+
+              has('certificateReissuance') &&
+              it('check certificateReissuance', () => {
+                if (expCertificateReissuance) {
+                  expect(output).to.have.nested.property('certificateReissuance')
```

Comment:
I see no need for the `.nested` chain occurrences when testing the `certificateReissuance` node. Removing it won't change the test result.

Reply:
see https://github.com/corona-warn-app/cwa-app-ccl/pull/32/commits/0a9e5e449359c794c2c05472cd07ee01c80d9865